### PR TITLE
[MOB-5969] stop when logged out

### PR DESCRIPTION
**Pull request by roninopf**
## 🔹 Jira Ticket(s)

* [MOB-5969](https://iterable.atlassian.net/browse/MOB-5969)

## ✏️ Description

* stop manager from retrieving when logged out
* slight reorganization and clarity

[MOB-5969]: https://iterable.atlassian.net/browse/MOB-5969?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: swift-sdk/Internal/EmbeddedMessagingManager.swift [hunk 1]
```diff
@@ -16,16 +16,12 @@ class EmbeddedMessagingManager: NSObject, IterableEmbeddedMessagingManagerProtoc
         self.dateProvider = dateProvider
         
         super.init()
-        
-        initObservers()
-        initAutoFetchTimer()
     }
```
Comment (evantk91) [inline swift-sdk/Internal/EmbeddedMessagingManager.swift:19]: I believe we are missing a start call here

**Review by evantk91 — COMMENTED** [review]
Minor comment

File: swift-sdk/Internal/EmbeddedMessagingManager.swift (hunk 1 above)
Comment (roninopf) [reply swift-sdk/Internal/EmbeddedMessagingManager.swift:19]: It's done in `InternalIterableAPI`. It was already getting called, but that was doing nothing before this PR.

Comment (evantk91) [reply swift-sdk/Internal/EmbeddedMessagingManager.swift:19]: Oh gotcha at line 618

**Review by evantk91 — APPROVED** [review]
LGTM 👍

**Pull request merged**